### DISCUSSION TITLE
Excalidraw links

### DISCUSSION THREAD

**Discussion by paquitxuelo** [oleeskild/obsidian-digital-garden#132] [General]
Hi Ole,
Following your Digital Garden Docs, I’ve been able to publish via Netlify a static web. Many thanks for that.
I’ve found a little problem that I haven’t been able to solve alone yet. It has to do with Excalidraw for Obsidian. I achieve publishing the file nicely in any form (being excalidraw, png, or svg). Then problem arises when some element of the drawing has attached a link to another note: something gets lost in the process of publishing and the static site can’t find the correct file. In the excalidraw for Obsidian works flawlessly (so indeed the element links to the desired note) but the static page tries to find it somewhere else and fails.
Could you help me with that? Many thanks in advance.
Francesc

**Comment by oleeskild** [comment]
This is simply because it isn't really supported. I'm using the official [excalidraw](https://github.com/excalidraw/excalidraw) package to render excalidrawings. The note linking is something custom that the excalidraw obsidian plugin has implemented, and isn't really supported in the official package. Perhaps I'll implement it some day, but I suspect it would require a lot of work so it is currently not a priority. But thanks for the question. I will add an explanation of this in the docs, so that it doesn't confuse others as well.

**Comment by paquitxuelo** [comment]
Hi Ole,

I found a workaround to fix that issue. Maybe not so elegant, but it works.

  1.  Within Excalidraw Obsidian (.excalidraw file extension), when introducing the link address, use the actual address of the final Netlify site, for instance:

https://[your_repository _name].netlify.app/[some_Obsidian_note_name]

  1.  Save that file as SVG.
  2.  Transclude this image where you want to publish it
  3.  Publish that file with your dg-publish: true
  4.  Done
I guess my explanation is not the greatest, but I hope you will be able to decipher. If it is not the case, please, let me know it and I will try it again.
Thanks for your magnificent work,
Francesc

De: Ole Eskild ***@***.***>
Enviat: dimecres, 18 de gener de 2023 15:38
Per a: ***@***.***>
A/c: ***@***.***>; ***@***.***>
Tema: Re: [oleeskild/obsidian-digital-garden] Excalidraw links (Discussion #132)


This is simply because it isn't really supported. I'm using the official excalidraw<https://github.com/excalidraw/excalidraw> package to render excalidrawings. The note linking is something custom that the excalidraw obsidian plugin has implemented, and isn't really supported in the official package. Perhaps I'll implement it some day, but I suspect it would require a lot of work so it is currently not a priority. But thanks for the question. I will add an explanation of this in the docs, so that it doesn't confuse others as well.
